src/common/utils/consonants.ts

```diff
@@ -4,22 +4,22 @@ import { ApplicationStatus } from '../../@types/__generated__/globalTypes';
 type ApplicationStatusType = {
```

Comment:
Should we finally rename this file to `constants.ts`? This would be a time as good as any 😛

Reply:
Oh, great catch 😅 
Renamed!